src/vos/vos_iterator.c

```diff
@@ -666,18 +657,21 @@ vos_iterate_internal(vos_iter_param_t *param, vos_iter_type_t type,
 	}
 	read_time = dtx_is_valid_handle(dth) ? dth->dth_epoch : 0 /* unused */;
 probe:
-	if (!daos_anchor_is_zero(anchor))
-		probe_anchor = anchor;
-
-	rc = vos_iter_probe(ih, probe_anchor);
+	rc = vos_iter_probe(ih, anchor);
```

Comment:
Why abort isn't handled?

Reply:
It is handled.  It exits.  It's basically similar to returning 1